app/common/data/interfaces/collections.py

```diff
@@ -23,8 +23,16 @@ def create_collection_schema(*, name: str, user: User, grant: Grant, version: in
     return schema
 
 
-def get_collection_schema(collection_id: UUID4) -> CollectionSchema:
-    return db.session.get_one(CollectionSchema, collection_id)
+def get_collection_schema(collection_id: UUID4, version: int | None = None) -> CollectionSchema:
```

Comment:
Can we add a docstring that says this defaults to the latest version if `version` is `None` please?